project/searching.py

```diff
@@ -1,30 +1,35 @@
-# STRETCH: implement Linear Search				
+# STRETCH: implement Linear Search
 def linear_search(arr, target):
-  
-  # TO-DO: add missing code
 
-   return -1   # not found
+    # TO-DO: add missing code
+    for index, val in enumerate(arr):
+        if val == target:
+            return f'We found the item {val} at index {index}'
+    return -1   # not found
 
```

Comment:
I like that you implemented :   return f'We found the item {val} at index {index}'